tonic-build/README.md

```diff
@@ -68,21 +95,23 @@ And a bunch of Google proto files in structure will be like this:
 │               └── schema.proto
 ```
 
-Then we can generate Rust code via this setup in our `build.rs`
-```rust
-fn main() {
+Then we can generate Rust code via this setup in our `build.rs`:
+
+```norun
```

Comment:
Not sure if having a worse highlighting in the readme is acceptable.

The alternative would be to copy the readme to `lib.rs`.
I cannot change it to `rust` or `rust,norun` as both throw the same error that the protos are not present.